src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/ControlFlowInstructions.cs

```diff
@@ -767,12 +767,12 @@ private ThrowInstruction(bool hasResult, bool isRethrow)
 
         public override int Run(InterpretedFrame frame)
         {
-            var ex = (Exception)frame.Pop();
+            var ex = frame.Pop();
```

Comment:
Even if with a style strongly in favour of `var` it's at odds with multiple similar lines in that file, so should still be explicit. Fixed.